Docker Desktop release notes 2.5.2 (#11791)
docker/docker.github.io: docker-for-mac/edge-release-notes.md

For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
[What to know before you install](install.md#what-to-know-before-you-install).

## Docker Desktop Community 2.5.2
2020-11-24

> [Download](https://desktop.docker.com/mac/edge/50063/Docker.dmg)

> ### New

Reviewer: ```suggestion
### New

```

- Use of three digit version number.

> - Starting from now new updates will be much smaller because they will be applied using delta patches.

Reviewer: ```suggestion
- Starting with Docker Desktop 2.5.2, updates will be much smaller as they will be applied using delta patches.
```

Any rough idea as to how much smaller the new updates will be?

Author: I'm not quite sure, but it will depend on the patches made. Although I don't think they go over 100MB (considering that Docker for Mac is aroun ~500MB)

Reviewer 2: I don't think we want to promise any numbers at this stage. We'll need to get some experience first.

> - Support macOS Big Sur with Intel processors.

Reviewer: ```suggestion
- Support for MacOS 11.0 (Big Sur) with Intel processors.
```

### Bug fixes and minor changes

> - Re-enable the experimental SOCKS proxy. Fixes [docker/for-mac#5048](https://github.com/docker/for-mac/issues/5048).

Reviewer 2: Re-enabled

Author: done!

For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
[What to know before you install](install.md#what-to-know-before-you-install).

## Docker Desktop Community 2.5.2
2020-11-24

> [Download](https://desktop.docker.com/mac/edge/50063/Docker.dmg)

### New

- Use of three digit version number.
- Starting with Docker Desktop 2.5.2, updates will be much smaller as they will be applied using delta patches.
- Support for MacOS 11.0 (Big Sur) with Intel processors.

### Bug fixes and minor changes

- Re-enabled the experimental SOCKS proxy. Fixes [docker/for-mac#5048](https://github.com/docker/for-mac/issues/5048).

> - Fixed an unexpected EOF error when trying to start a non-existing container with `-v /var/run/docker.sock:`. See [docker/for-mac#5025](https://github.com/docker/for-mac/issues/5025).

Reviewer 2: I think this applies on Windows too even though it was reported on for-mac, we should copy it across.

Author: Right!
